web-components/src/components/tooltip/InfoIconTooltip.tsx

```diff
@@ -0,0 +1,19 @@
+import InfoIcon from '../icons/InfoIcon';
+import InfoIconOutlined from '../icons/InfoIconOutlined';
+import InfoTooltip from './InfoTooltip';
+
+interface Props {
+  title: string;
+  outlined?: boolean;
+}
+
+export default function InfoIconTooltip({
```

Comment:
Should we add a case in `tooltip.stories.tsx` for this component?

Reply:
Of course, I missed it, thanks!